src/site/content/en/blog/notification-triggers/index.md

```diff
@@ -0,0 +1,259 @@
+Origin trials allow you to try new features and give feedback on their usability, practicality, and
+effectiveness, both to us, and to the web standards community. For more information, see the
+[Origin Trials Guide for Web Developers](https://googlechrome.github.io/OriginTrials/developer-guide.html).
+
+Request a [token](https://developers.chrome.com/origintrials/) for your origin. Add the token to
+your pages, there are two ways to provide this token on any pages in your origin: Add an
+`origin-trial` `<meta>` tag to the `head` of any page. For example, this may look something like:
+`<meta http-equiv="origin-trial" content="TOKEN_GOES_HERE">` If you can configure your server, you
+can also provide the token on pages using an `Origin-Trial` HTTP header. The resulting response
+header should look something like: `Origin-Trial: TOKEN_GOES_HERE`
+
+### Feature detection
+
+You can find out if the browser supports Notification Triggers by checking for the existence of the
+`showTrigger` property.
+
+```js
+if ("showTrigger" in Notification.prototype) {
+  /* Notification Triggers supported */
+}
+```
+
+### Scheduling a notification
+
+Scheduling a notification is similar to showing a regular push notification, with the difference
+being that you need to pass a `showTrigger` condition property with a `TimestampTrigger` object as
+the value to the notification's options object.
+
+```js
+const createScheduledNotification = async (tag, title, timestamp) => {
+  const registration = await navigator.serviceWorker.getRegistration();
+  registration.showNotification(title, {
+    tag: tag,
+    body: "This notification was scheduled 30 seconds ago",
+    showTrigger: new TimestampTrigger(timestamp + 30 * 1000)
+  });
+};
+```
+
+{% Aside %}
+  On desktop, notification triggers fire only if Chrome is running.
+  On Android, they fire regardless.
+{% endAside %}
+
+### Canceling a scheduled notification
+
+To cancel scheduled notifications, first request a list of all notifications that match a
+certain tag through `ServiceWorkerRegistration.getNotifications()`. Note that you need to pass the
+`includeTriggered` flag for scheduled notifications to be included in the list.
+
+```js
+const cancelScheduledNotification = async (tag) => {
+  const registration = await navigator.serviceWorker.getRegistration();  
+  const notifications = await registration.getNotifications({
+    tag: tag,
+    includeTriggered: true,
+  }); 
+  notifications.forEach((notification) => notification.close());
+};
+```
+
+### Debugging
+
+You can use Chrome DevTools' Notifications debug service on the Application tab. To start debugging,
+press the small "Start recording events" button in the top-left corner (the circle), or hit
+'Command/Control' + 'E'. This allows you to both examine scheduled notifications, as well as see
+displayed and closed notifications. The screenshots below show this in action.
```

Comment:
We usually use the `kbd` HTML tag for annotating keyboard characters. I will fix this. We can also link to a DevTools doc on the topic.

Reply:
Thanks, I literally learned about `kbd` on Nov. 2 via [this tweet](https://twitter.com/samantha_ming/status/1190704279264194560).